recipes/kanga/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "kanga" %}
+{% set version = "0.0.17" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0e659f7c54530565bbae103844e5b736b7eb6e12ab6b07039023a0a5d6ea7375
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib >=3.3.3
```

Unless you need a PyQt-based GUI, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base >=3.3.3
```

---

> Unless you need a PyQt-based GUI, better to depend on `matplotlib-base`:

Thank you very much for the tip @dopplershift as I wasn't aware of the difference between `matplotlib-base` and `matplotlib`. I made the change you suggested.